lib/evmone/analysis.cpp

```diff
@@ -11,6 +11,19 @@ namespace evmone
 {
 namespace
 {
+struct block_analysis
+{
+    int gas_cost = 0;
+
+    int stack_req = 0;
+    int stack_max_growth = 0;
+    int stack_change = 0;
+
+    size_t first_instruction_index = 0;
```

Comment:
Maybe it's `begin_block_index`?
As I understand it can be either `OPX_BEGINBLOCK` or `OP_JUMPDEST`, but "first instructions" sounds a bit like a first real instruction in the block

Reply:
This is effectively always `OPX_BEGINBLOCK` as we skip `OP_JUMPDEST` from instruction table. Anyway, I'll rename it and document more.

Reply:
Renamed.